Purchases/Public/RCLogLevel.h

```diff
@@ -0,0 +1,21 @@
+//
+//  RCLogLevel.h
+//  Purchases
+//
+//  Created by Andrés Boedo on 6/3/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+// These must match the values in LogLevel.swift exactly
+typedef NS_ENUM(NSInteger, RCLogLevel) {
+    RCLogLevelDebug = 0,
```

Comment:
i meant here, rather than setting the value 0,1,2,3 (or in addition to, if you use those values), you could have the internal log level. just creates an easier mapping

Reply:
oh, that's much smarter than what I was doing, I'll update. thanks!

Reply:
oh, right, just remembered why we can't do that 😢 
You can't forward-declare enum values in objective-c, so there's no way to reference the values from this header file, sadly.